variables.tf

```diff
@@ -0,0 +1,84 @@
+variable "aws_region" {
+  type        = string
+  description = "AWS Region"
+  default     = "eu-central-1"
```

Comment:
idk if we should set a default here, as we never do set a default aws_region in other modules. Also if we did set one, i'd rather use us-east-1 because it is amazons default region